core/src/main/kotlin/org/evomaster/core/problem/rest/RestCallAction.kt

```diff
@@ -37,7 +37,8 @@ class RestCallAction(
          * by a POST, where the POST itself might use a location for
          * path coming from a previous POST
          */
-        var locationId: String? = null
+        var locationId: String? = null,
+        var produces: MutableList<String> = mutableListOf()
```

Comment:
why is this mutable?

Reply:
Not sure. I changed it to List. It will be visible once I push the changes.